common/src/main/scala/explore/components/forms/EnumSelect.scala

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.components.forms
+
+import cats.implicits._
+import cats.Show
+import cats.effect.IO
+import gem.util.Enumerated
+import japgolly.scalajs.react._
+import japgolly.scalajs.react.vdom.html_<^._
+import react.common.ReactProps
+import react.semanticui.modules.dropdown._
+import scala.scalajs.js.JSConverters._
+import crystal.react.io.implicits._
+
+/**
+  * Produces a dropdown menu, similar to a combobox
+  */
+final case class EnumSelect[A: Enumerated: Show](
```

Comment:
I guess this will go to gsp-ui

Reply:
Yes, we shouldn't have duplicate code in different projects (this is copied from `ocs3`).